src/lithopscloud/modules/code_engine/defaults.yaml

```diff
@@ -1,11 +1,10 @@
 lithops:
   backend: code_engine
-  storage: ibm_cos
+  storage: ""
```

Comment:
why is it not ibm_cos by default anymore?

Reply:
Future-proofing for a time where more storage backends will be made available. At the end of the workflow, cos.py simply sets the storage value, same as any additional storage backends, that will be added in the future, will do.